Gems/EMotionFX/Code/EMotionFX/Rendering/Common/RotateManipulator.cpp

```diff
@@ -405,45 +409,55 @@ namespace MCommon
 
             // the intersection variables
             AZ::Vector3 intersectA, intersectB;
+            AZ::Vector3 rayReciprocal = mousePosRay.GetDirection().GetReciprocal();
+            AZ::Vector3 normalizedDirection;
+            auto intersectRayDirectionAxis =
+                [=](AZ::Aabb& aabb, const MCore::Ray& ray, const AZ::Vector3& dirRCP, AZ::Vector3& normalizedDirection)
+            {
+                float start = 0, end = 0;
```

Comment:
uber nit: Please initialize with `0.0f` - makes it easier to read at a glance (and there is potentially an incredibly minor performance saving, but readability is the far more important thing in this case)